common/app/conf/Filters.scala

```diff
@@ -76,11 +76,11 @@ object Filters {
   // NOTE - order is important here, Gzipper AFTER CorsVaryHeaders
   // which effectively means "JsonVaryHeaders goes around Gzipper"
   def common: List[EssentialFilter] = List(
+    new RequestLoggingFilter,
```

Comment:
I moved this because it wasn't actually logging/timing the whole request/response chain.  Perhaps that was intended, but it wasn't quite realistic.